docs/uart.rst

```diff
@@ -10,17 +10,19 @@ a serial interface.
 Functions
 =========
 
-.. method:: init(baudrate=9600, bits=8, parity=None, stop=1, pins=None)
+.. method:: init(baudrate=9600, bits=8, parity=None, stop=1, \*, tx=None, rx=None)
```

Comment:
That's to mark the tx and rx arguments as keyword-only -- that is, they can't be specified as positional arguments.